frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/multiple-pool.spec.ts

```diff
@@ -0,0 +1,92 @@
+import { checkErrors } from '../../../integration-tests-cypress/support';
+import { NS, CLUSTER_STATUS } from '../utils/consts';
+import {
+  cancelAction,
+  confirmAction,
+  createStorageClassButton,
+  emptyStateBody,
+  storagePoolDropdown,
+  storagePool,
+} from '../views/multiple-pool';
+
+describe('Test ceph pool creation', () => {
+  
+  // Ceph status var
+  let cephStatus:any;
+
+  // Pool var
+  const poolName: string = 'example.com';
+  const replicaCount: string = '2';
+  const provisioner: string = 'openshift-storage.rbd.csi.ceph.com';
+
+  before(() => {
+    cy.login();
+    cy.visit('/');
+    // cy.install();
+
+    // Check ceph cluster status
+    cy.exec(`oc get cephCluster -n ${NS} -o json`).then((res) => {
+      const cephValue = JSON.parse(res.stdout.toString());
```

Comment:
```suggestion
      const cephValue = JSON.parse(res.stdout);
```
I don't think if `toSting()` is required.

Reply:
ack